lineage/tHMM.py

```diff
@@ -97,8 +97,12 @@ def get_AIC(self, LL):
         if self.fT is None:
             degrees_of_freedom += self.num_states * (self.num_states - 1)
 
-        for ii in range(self.num_states):
-            degrees_of_freedom += self.estimate.E[ii].dof()
+        if atonce: # assuming we are fitting all 4 concentrations at once and we have cell cycle phase specific distributions.
+            for ii in range(self.num_states):
```

Comment:
Do you need a loop here? Can't you just say `degrees_of_freedom = self.num_states * 4.5`?